### ISSUE TITLE
FluxStore Woo - Greater Than From Interger Error

### ISSUE THREAD

**Issue by buddiezweb** [inspireui/support#3507]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
When I Run the App and then click on any product variation it show this error
+ Product & version: Fluxstore WooCommerce - Flutter E-commerce | FluxStore-woo 1.7.4
+ Flutter (or React Native) version: 1.17.2
+ Testing Device/Simulator: iOS iphone 11 and Pixel 2
+ Screenshot issues (drag the file to attach here): 
<img width="1704" alt="Screenshot 2020-05-31 at 6 10 30 PM" src="https://user-images.githubusercontent.com/8961536/83352698-1a856780-a36b-11ea-811a-4a5d89c25219.png">

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

**_Important Note:_**
- We will close the ticket if missing the proof of purchase on step 2. The screenshot purchase attaches here will be invalid.
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by philipvu30** [comment]
Hi,

Replace this file in the path lib/screens/detail/product_variant.dart
[product_variant.dart.zip](https://github.com/inspireui/support/files/4709151/product_variant.dart.zip)

**Comment by buddiezweb** [comment]
Now it Shows this error.
<img width="1742" alt="Screenshot 2020-06-01 at 10 23 58 AM" src="https://user-images.githubusercontent.com/8961536/83377078-181a2080-a3f2-11ea-806a-e732796fb56b.png">

Launching lib/main.dart on iPhone 11 in debug mode...
Running pod install...                                          






Compiler message:
lib/widgets/product/cart_item.dart:64:36: Error: The method 'QuantitySelection' isn't defined for the class 'ShoppingCartRow'.
 - 'ShoppingCartRow' is from 'package:fstore/widgets/product/cart_item.dart' ('lib/widgets/product/cart_item.dart').
Try correcting the name to the name of an existing method, or defining a method named 'QuantitySelection'.
                            child: QuantitySelection(
                                   ^^^^^^^^^^^^^^^^^










lib/widgets/frameworks/product_variant_mixin.dart:205:22: Error: The method 'QuantitySelection' isn't defined for the class 'ProductVariantMixin'.
 - 'ProductVariantMixin' is from 'package:fstore/widgets/frameworks/product_variant_mixin.dart' ('lib/widgets/frameworks/product_variant_mixin.dart').
Try correcting the name to the name of an existing method, or defining a method named 'QuantitySelection'.
              child: QuantitySelection(
                     ^^^^^^^^^^^^^^^^^
lib/widgets/frameworks/woocommerce/woo_variant_mixin.dart:106:13: Error: The method 'BasicSelection' isn't defined for the class 'WooVariantMixin'.
 - 'WooVariantMixin' is from 'package:fstore/widgets/frameworks/woocommerce/woo_variant_mixin.dart' ('lib/widgets/frameworks/woocommerce/woo_variant_mixin.dart').
Try correcting the name to the name of an existing method, or defining a method named 'BasicSelection'.
            BasicSelection(
            ^^^^^^^^^^^^^^
Running pod install...                                              6.3s
Running Xcode build...
Xcode build done.                                           45.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED;
    ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose 'requestReview' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
                                 ^~~~~~~~~~~~~~
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:15:6: warning: 'SKStoreReviewController' is only available on iOS 10.3 or newer [-Wunguarded-availability]
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    In module 'StoreKit' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/StoreKit.framework/Headers/SKStoreReviewController.h:14:110: note: 'SKStoreReviewController' has been marked as being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0
    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14)) API_UNAVAILABLE(watchos) __TVOS_PROHIBITED @interface SKStoreReviewController : NSObject
                                                                                                                 ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.1+2/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose 'SKStoreReviewController' in an @available check to silence this warning
        [SKStoreReviewController requestReview];
         ^~~~~~~~~~~~~~~~~~~~~~~
    2 warnings generated.
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:195:43: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message handling. [-Wdeprecated-declarations]
    - (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                                              ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:307:24: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message handling. [-Wdeprecated-declarations]
        didReceiveMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                           ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
    __deprecated_msg(
    ^
    In module 'UIKit' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
            #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                          ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:141:30: warning: 'shouldEstablishDirectChannel' is deprecated: FCM direct channel is deprecated, please use APNs channel for downstream message delivery. [-Wdeprecated-declarations]
        [FIRMessaging messaging].shouldEstablishDirectChannel = true;
                                 ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:283:56: note: 'shouldEstablishDirectChannel' has been explicitly marked deprecated here
    @property(nonatomic) BOOL shouldEstablishDirectChannel DEPRECATED_MSG_ATTRIBUTE(
                                                           ^
    In module 'UIKit' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/usr/include/AvailabilityMacros.h:182:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
                #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                                   ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:196:52: warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [self didReceiveRemoteNotification:remoteMessage.appData];
                                                       ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:306:1: warning: implementing deprecated method [-Wdeprecated-implementations]
    - (void)messaging:(FIRMessaging *)messaging
    ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:253:1: note: method 'messaging:didReceiveMessage:' declared here
    - (void)messaging:(FIRMessaging *)messaging
    ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.15/ios/Classes/FLTFirebaseMessagingPlugin.m:308:63: warning: 'appData' is deprecated [-Wdeprecated-declarations]
      [_channel invokeMethod:@"onMessage" arguments:remoteMessage.appData];
                                                                  ^
    In module 'FirebaseMessaging' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:72:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/FirebaseMessaging/Firebase/Messaging/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
    @property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                                 ^
    In module 'UIKit' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
    In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
    In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
    In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.5.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
            #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                               ^
    6 warnings generated.
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:80:37: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd
                                        ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:87:37: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd
                                        ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:95:49: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdDidReceiveAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:100:44: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdDidOpen:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                               ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:104:52: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdDidStartPlaying:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                       ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:108:55: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdDidCompletePlaying:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                          ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:112:45: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdDidClose:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:117:57: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    - (void)rewardBasedVideoAdWillLeaveApplication:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                            ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:39:28: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
        [GADRewardBasedVideoAd sharedInstance].delegate = self;
                               ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:56:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
      [[GADRewardBasedVideoAd sharedInstance] loadRequest:[factory createRequest]
                              ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:61:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
      [[GADRewardBasedVideoAd sharedInstance]
                              ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:66:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
      [[GADRewardBasedVideoAd sharedInstance] setUserIdentifier:userIdentifier];
                              ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:70:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
      [[GADRewardBasedVideoAd sharedInstance] setCustomRewardString:customRewardString];
                              ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:75:66: warning: implicit conversion loses integer precision: 'FLTRewardedVideoAdStatus' to 'int' [-Wshorten-64-to-32]
          (NSString *)rewardedStatusToString[[NSNumber numberWithInt:_rewardedStatus]];
                                             ~                       ^~~~~~~~~~~~~~~
    14 warnings generated.
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:71:13: warning: 'testDevices' is deprecated: Use GADMobileAds.sharedInstance.requestConfiguration.testDeviceIdentifiers. [-Wdeprecated-declarations]
        request.testDevices = testDevices;
                ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:88:71: note: 'testDevices' has been explicitly marked deprecated here
    @property(nonatomic, copy, nullable) NSArray<NSString *> *testDevices GAD_DEPRECATED_MSG_ATTRIBUTE(
                                                                          ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:91:15: warning: 'birthday' is deprecated [-Wdeprecated-declarations]
          request.birthday =
                  ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:95:55: note: 'birthday' has been explicitly marked deprecated here
    @property(nonatomic, copy, nullable) NSDate *birthday GAD_DEPRECATED_ATTRIBUTE;
                                                          ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                    ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:106:19: warning: 'gender' is deprecated [-Wdeprecated-declarations]
              request.gender = genderValue;
                      ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:92:47: note: 'gender' has been explicitly marked deprecated here
    @property(nonatomic, assign) GADGender gender GAD_DEPRECATED_ATTRIBUTE;
                                                  ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                    ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:116:14: warning: 'tagForChildDirectedTreatment:' is deprecated:  use [GADMobileAds.sharedInstance.requestConfiguration tagForChildDirectedTreatment] [-Wdeprecated-declarations]
        [request tagForChildDirectedTreatment:childDirected.boolValue];
                 ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:126:5: note: 'tagForChildDirectedTreatment:' has been explicitly marked deprecated here
        GAD_DEPRECATED_MSG_ATTRIBUTE(
        ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    4 warnings generated.
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTMobileAd.m:39:17: warning: 'configureWithApplicationID:' is deprecated: Use [GADMobileAds.sharedInstance startWithCompletionHandler:] [-Wdeprecated-declarations]
      [GADMobileAds configureWithApplicationID:appId];
                    ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:161:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADMobileAds.h:84:5: note: 'configureWithApplicationID:' has been explicitly marked deprecated here
        GAD_DEPRECATED_MSG_ATTRIBUTE("Use [GADMobileAds.sharedInstance startWithCompletionHandler:]");
        ^
    In module 'GoogleMobileAds' imported from /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Headers/Public/Firebase/Firebase.h:161:
    /Users/rajhussain/Desktop/BuddiezWeb/Redantz/Mobile/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                           ^
    /Users/rajhussain/Development/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+2/ios/Classes/FLTMobileAd.m:97:79: warning: implicit conversion loses integer precision: 'FLTMobileAdStatus' to 'int' [-Wshorten-64-to-32]
      NSString *statusString = (NSString *)statusToString[[NSNumber numberWithInt:_status]];
                                                          ~                       ^~~~~~~
    2 warnings generated.

    Compiler message:
    lib/widgets/product/cart_item.dart:64:36: Error: The method 'QuantitySelection' isn't defined for the class 'ShoppingCartRow'.
     - 'ShoppingCartRow' is from 'package:fstore/widgets/product/cart_item.dart' ('lib/widgets/product/cart_item.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'QuantitySelection'.
                                child: QuantitySelection(
                                       ^^^^^^^^^^^^^^^^^
    lib/widgets/frameworks/product_variant_mixin.dart:205:22: Error: The method 'QuantitySelection' isn't defined for the class 'ProductVariantMixin'.
     - 'ProductVariantMixin' is from 'package:fstore/widgets/frameworks/product_variant_mixin.dart' ('lib/widgets/frameworks/product_variant_mixin.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'QuantitySelection'.
                  child: QuantitySelection(
                         ^^^^^^^^^^^^^^^^^
    lib/widgets/frameworks/woocommerce/woo_variant_mixin.dart:106:13: Error: The method 'BasicSelection' isn't defined for the class 'WooVariantMixin'.
     - 'WooVariantMixin' is from 'package:fstore/widgets/frameworks/woocommerce/woo_variant_mixin.dart' ('lib/widgets/frameworks/woocommerce/woo_variant_mixin.dart').
    Try correcting the name to the name of an existing method, or defining a method named 'BasicSelection'.
                BasicSelection(
                ^^^^^^^^^^^^^^
    Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
    build failed.
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 11.

**Comment by philipvu30** [comment]
Hi,

Can you try again with this file?
[product_variant.dart.zip](https://github.com/inspireui/support/files/4709535/product_variant.dart.zip)

**Comment by buddiezweb** [comment]
Now when I try to get an archive from xcode I get this error.
<img width="1792" alt="Screenshot 2020-06-01 at 8 21 44 PM" src="https://user-images.githubusercontent.com/8961536/83421337-8dfba780-a445-11ea-910e-0ffac99a80ad.png">

**Comment by philipvu30** [comment]
Hi,

Please refer to this [document](https://documentation.onesignal.com/docs/flutter-sdk-setup) to setup OneSignal for the IOS.

**Comment by buddiezweb** [comment]
I don't want to use onesignal I want to remove it so how do I do it

**Comment by buddiezweb** [comment]
I deleted the Extension and now I am having this error.

<img width="1792" alt="Screenshot 2020-06-03 at 9 27 37 AM" src="https://user-images.githubusercontent.com/8961536/83594357-a28e8b80-a57c-11ea-82f2-1e7ef0155a9d.png">

**Comment by philipvu30** [comment]
Hi,

Would you please run the app on Android Studio?

**Comment by philipvu30** [comment]
Hi,

We just updated the FluxStore from 1.7.4 to 1.7.5. If you have access to our Repository, please update it and see if the issue is solved. 
Otherwise, you have to wait a few days before we upload a new version to CodeCanyon.

My apologies for this inconvenience.

**Comment by buddiezweb** [comment]
I have Found a bug or issue. When we add a product by clicking on buy now button and then we clear the cart and then we are on a screen were it shows to click on start shopping but we are unable to.

**Comment by philipvu30** [comment]
Hi,

I cannot reproduce the issue as it is fine on my side. Would you mind including an error log or error image?

Thank you.

**Comment by minhcasi** [comment]
@buddiezweb - pls use the latest release `1.7.5` to fix this issue - thanks

**Comment by buddiezweb** [comment]
How can I use Google Analytics.

**Comment by buddiezweb** [comment]
Do I have to Update the app in play store and app store every time I need to update the images or I change the config_en.dart file

**Comment by buddiezweb** [comment]
Where can I update the privacy and terms page the is no link option.

**Comment by buddiezweb** [comment]
@philipvu30 @minhcasi  How can I enable COD in App and also it should show the cod charges and I am unable to see tax rates in the app it should show.

**Comment by philipvu30** [comment]
Hi,

Sorry for the delayed response as we rarely work on weekends.
> How can I use Google Analytics.

Currently, we don't have any guide on this feature.
> Do I have to Update the app in play store and app store every time I need to update the images or I change the config_en.dart file

You can use the upload config.json of MStore plugin so you don't have to update the app via AppStore and PlayStore every time your images change. But with this config, you cannot use the multi-languages feature.
> Where can I update the privacy and terms page the is no link option.

In lib/screens/settings/settings.dart
![image](https://user-images.githubusercontent.com/54442328/84014348-5c825f00-a9a4-11ea-8c66-a90893be3705.png)
>How can I enable COD in App and also it should show the cod charges and I am unable to see tax rates in the app it should show.

Sorry, we have not implemented this feature yet.

**Comment by buddiezweb** [comment]
I need to show the customer the Tax(GST) where and how can I show that.

**Comment by philipvu30** [comment]
Hi,

I don't know what GST tax you are referring to? Currently, the app is getting the order total_tax from woocommerce order properties.

![image](https://user-images.githubusercontent.com/54442328/84617255-95688980-aef8-11ea-86f2-fcea4eaffb49.png)

**Comment by buddiezweb** [comment]
The App was not showing the total tax also.

**Comment by philipvu30** [comment]
Hi,

Where do you want to display the total tax?

**Comment by buddiezweb** [comment]
At Check out page and is there any way i can sort products randomly on home screen.

**Comment by philipvu30** [comment]
Hi,

>At Check out page

The tax can only show after the order is made because it is one of the Order's properties. You'll need to implement this yourself. Sorry.

>is there any way i can sort products randomly on home screen.

I quite don't get what you mean. You can start investigating these 2 functions fetchProductsByCategory and fetchProductsLayout in lib/services/woo_commerce.dart and see if they can give you any idea.

Thanks.

**Comment by buddiezweb** [comment]
I want to sort the product in fetchProductsLayout randomly and the rate the app is not working in iOS devices its showing on simulator.

**Comment by anmolmajhail** [comment]
FluxStore - Woo - taxes are not showing there too- it is show stopper for the app to go live.
